qt/scientific_interfaces/Indirect/IndirectFitAnalysisTab.cpp

```diff
@@ -355,6 +355,9 @@ void IndirectFitAnalysisTab::endXChanged(double endX) {
  */
 void IndirectFitAnalysisTab::setConvolveMembers(bool convolveMembers) {
   m_fitPropertyBrowser->setConvolveMembers(convolveMembers);
+  // if convolve members is on, output members should also be on
```

Comment:
Possibly redundant comment

Reply:
This comment is explaining why we call `setOutputCompositeMembers` in the if loop that follows. If in the code we set convolved members to true, then we also need to set `OutputCompositeMembers` to true else no output will be produced.

Reply:
Understood